src/components/input/input.jsx

```diff
@@ -50,6 +50,9 @@ const Input = forwardRef(function Input(props, forwardedRef) {
     },
   }));
 
+  const describedBy = [`${props.id}Hint`];
+  if (props.tooltip) describedBy.push(`${props.id}-tooltip`);
```

Comment:
I think it's okay to have a `describedby` with a non-existent element. Since it does not find the element, I think it just skips it.

```suggestion
  describedBy.push(`${props.id}-tooltip`);
```

I would prefer the simpler code than the branching code. But first it's gotta work!

Reply:
9a2c0f8d